src/main/java/net/minecraftforge/common/ForgeMod.java

```diff
@@ -93,14 +95,21 @@
     public static final RegistryObject<Attribute> ENTITY_GRAVITY = ATTRIBUTES.register("entity_gravity", () -> new RangedAttribute("forge.entity_gravity", 0.08D, -8.0D, 8.0D).setSyncable(true));
 
     /**
-     * Reach Distance represents the distance at which a player may interact with the world.  The default is 4.5 blocks.  Players in creative mode have an additional 0.5 blocks of reach distance.
+     * Reach Distance represents the distance at which a player may interact with the world.  The default is 4.5 blocks.  Players in creative mode have an additional 0.5 blocks of reach distance.<br>
+     * See {@link IForgePlayer#getReachDistance} and {@link IForgePlayer#canInteractWith}
```

Comment:
Use the `@see` tag instead of `{@link}` for linking to other members outside of inline in a sentence. (You may also need to qualify the method references with the parameter types.) Also, remove the trailing `<br>`.
```suggestion
     * @see IForgePlayer#getReachDistance
     * @see IForgePlayer#canInteractWith
```